csharp/sample/Microsoft.ML.OnnxRuntime.InferenceSample/Program.cs

```diff
@@ -24,8 +24,13 @@ static void UseApi()
         {
             string modelPath = Directory.GetCurrentDirectory() + @"\squeezenet.onnx";
 
+            // Optional : Create session options and set the graph optimization level for the session
+            // Default graph optimization level is set to 1 which enables all basic optimizations (Such as redundant node removals)
+            // Level 2 enables all optimizations (Includes level 1 + more complex optimizations like node fusions)
```

Comment:
Can we remove the last 2 lines...just keep the first comment ? A  detailed description for SetSessionGraphOptimizationLevel() is available alongside the function in case  a user needs more information.